### DISCUSSION TITLE
antv x6为啥不能设置自动触发鼠标事件

### DISCUSSION THREAD

**Discussion by Songgexnl** [antvis/X6#3725] [Q&A]
我在画布中固定了一些位置，我从画布外开始向画布中拖拽，快到了预留的位置但是不会自动吸附，这里的吸附是吸附到我预留的位置，本打算写一个鼠标抬起事件，但是我拖拽进去之后，鼠标松开，事件没有触发，，但是一点击拖拽进去的元素这个事件就触发了，在官网没找自动触发鼠标事件，大佬给点解决办法
![image](https://github.com/antvis/X6/assets/80017602/5879153a-678d-4c7b-82eb-1f92cb6ad84e)
![image](https://github.com/antvis/X6/assets/80017602/5d9a7dac-0952-4464-98d6-38e5bdb84b93)

**Comment by NewByVector** [comment]
你是通过 dnd 插件来实现节点拖拽吗，如果是的话，可以通过监听 `node:added` 事件拿到放置节点的时机。

**Reply by Songgexnl** [reply]
是的，解决了，多谢大佬
